packages/bpk-component-map/stories.js

```diff
@@ -216,4 +316,5 @@ storiesOf('bpk-component-map', module)
         icon={<AlignedBusIconLg />}
       />
     </StoryMap>
-  ));
+  ))
+  .add('Stateful price markers', () => <StatefulBpkPriceMarker />);
```

Comment:
Nitpick, but we can probably just call this 'price markers' as it's the only example we have of them (i.e. no reason to call them out as being stateful).

```suggestion
  .add('Price markers', () => <StatefulBpkPriceMarker />);
```

Reply:
👍